api.planx.uk/modules/webhooks/service/analyzeSessions/operations.ts

```diff
@@ -6,13 +6,18 @@ import type { Operation } from "../sanitiseApplicationData/types.js";
 
 /**
  * ALLOW_LIST should stay in sync with
- * editor.planx.uk/src/pages/FlowEditor/lib/analyticsProvider
- * If appending to ALLOW_LIST please also update the
- * `submission_services_summary` view to extract it into it's own column.
+ *   editor.planx.uk/src/pages/FlowEditor/lib/analyticsProvider
+ * 
+ * If appending values to ALLOW_LIST please also update the
+ *  `analytics_summary` & `submission_services_summary` views to extract the value into its own column
+ * 
+ * Please also ensure your migration ends with `GRANT SELECT ON public.{VIEW_NAME} TO metabase_read_only`
+ *  so that Metabase picks up the new columns
  */
-
 const ALLOW_LIST = [
   "application.declaration.connection",
+  "application.information.harmful",
+  "application.information.sensitive",
```

Comment:
These are both yes/no questions - see here https://editor.planx.uk/doncaster/pre-application-advice

They additionally have extra "description" & "reason" free text inputs each, but those seem less chart-able & potentially sensitve info so I'm not including here - is that okay @zz-hh-aa ? Intention of analytics "allow lists" is to track generic, non-personally-identifiable stuff only in aggregate over all time. Doncaster of course will still get all answers with their submission data !

![Screenshot from 2024-10-07 16-33-54](https://github.com/user-attachments/assets/4515ba3d-f590-44a4-92db-5c494c6a2606)

Reply:
Yes I think that'll do just fine! Thanks Jess :)